schemas/hygiene/aep-hygiene-ops-record.schema.json

```diff
@@ -0,0 +1,100 @@
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/aep-hygiene-ops",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "AEP Hygiene Operation Request",
+  "description": "AEP Data Hygiene operations that instruct AEP to delete or modify records in a specified dataset and sandbox.",
+  "type": "object",
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "meta:extensible": false,
+  "meta:abstract": false,
+  "definitions": {
+    "hygieneOps": {
+      "properties": {
+        "xdm:batchTime": {
+          "title": "Time the batch was created",
+          "description": "Time the hygiene batch was created",
+          "type": "string",
+          "format": "date-time"
+        },
+        "xdm:action": {
+          "title": "Name of the hygiene operation to be applied by downstream consumers",
+          "description": "Name of the hygiene operation to be applied by downstream consumers.",
+          "type": "string",
+          "enum": ["delete_identity", "update_field"]
+        },
+        "xdm:targetDatasetId": {
+          "title": "Target Dataset ID specified by the user to apply hygiene operations",
+          "description": "May be \"*\", but only if operationType = \"delete_identity\". Otherwise a valid value is required.",
+          "type": "string"
+        },
+        "xdm:operation": {
+          "title": "Payload of the hygiene op request",
+          "description": "Contents of this payload provides identities to delete, or records to update",
+          "type": "object",
+          "properties": {
+            "xdm:delete_identity": {
+              "title": "Specifies rows to delete for matching identities",
+              "description": "Specifies rows to delete for matching identities",
+              "type": "object",
+              "properties": {
+                "xdm:identities": {
+                  "title": "Set of primary identities",
+                  "description": "Set of primary identities",
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "additionalProperties": false,
+                    "properties": {
+                      "xdm:namespace": {
+                        "title": "The identity namespace",
+                        "description": "The namespace associated with the xdm:id attribute.",
+                        "$ref": "https://ns.adobe.com/xdm/context/namespace"
+                      },
+                      "xdm:id": {
+                        "title": "Experience identifier",
+                        "type": "string",
+                        "description": "Experience identifier"
+                      }
+                    }
+                  }
+                }
+              }
+            },
+            "xdm:update_field": {
```

Comment:
Please rename it to "updateField" as we do not allow/recommend "_" or "-" in propertynames